typo3-ter/cal/2009-12-01.yaml

```diff
@@ -0,0 +1,8 @@
+title:     "TYPO3-SA-2009-019: Blind SQL Injection vulnerability in extension Calendar Base (cal)"
+link:      http://typo3.org/teams/security/security-bulletins/typo3-extensions/typo3-sa-2009-019/
+cve:
+branches:
+    master:
+        time:     2009-12-01 12:00
+        versions: [<=1.2.0]
```

Comment:
The version number containing the fix is not noted in the of the TYPO3 project's security bulletins. It would be really a lot of work to identify those release numbers. Therefore I would be happy if ``<=`` would also do the job.